module/task.py

```
@@ -155,25 +161,31 @@ def sort_skills_into_categories(skills):
-        if category in sorted_skills:
-            if name not in sorted_skills[category]:
-                sorted_skills[category].append(name)
+            if float(score) < 0.5:
+                continue
+                
+            matched_skills += 1
+            if float(score) >= 0.8:
+                high_confidence_matches += 1
+                
+            if category in sorted_skills:
+                if name not in sorted_skills[category]:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)